### PR TITLE
Feat: describe personal token

### PR DESCRIPTION
**Pull request by maxwelbm**

### CODE REVIEW

File: messages/personal_token/messages.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package personal_token
+
+const (
+	USAGE                      = "personal-token"
+	SHORT_DESCRIPTION_DESCRIBE = "Describe a Personal Token"
+	LONG_DESCRIPTION_DESCRIBE  = "Describe a Personal Token to be used for authentication and security"
+	ASK_PERSONAL_TOKEN_ID      = "Enter the ID of the personal token is linked to:"
+	FLAG_PERSONAL_TOKEN_ID     = "Unique identifier for an personal token"
```
Comment (rodrigomaria) [inline messages/personal_token/messages.go:8]:
```suggestion
	FLAG_PERSONAL_TOKEN_ID     = "Unique identifier for a personal token"
```

File: messages/personal_token/messages.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package personal_token
+
+const (
+	USAGE                      = "personal-token"
+	SHORT_DESCRIPTION_DESCRIBE = "Describe a Personal Token"
+	LONG_DESCRIPTION_DESCRIBE  = "Describe a Personal Token to be used for authentication and security"
+	ASK_PERSONAL_TOKEN_ID      = "Enter the ID of the personal token is linked to:"
```
Comment (PatrickMenoti) [inline messages/personal_token/messages.go:7]: "Enter the Personal Token's ID:"


-------
this is to follow the pattern of the other describe commands

File: pkg/api/personal_token/get.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+	sdk "github.com/aziontech/azionapi-go-sdk/personal_tokens"
+)
+
+func (c *Client) Get(ctx context.Context, personalTokenID string) (*sdk.PersonalTokenResponseGet, error) {
+	logger.Debug("Get Personal Token")
+	resp, httpResp, err := c.apiClient.PersonalTokenApi.GetPersonalToken(ctx, personalTokenID).Execute()
+	if err != nil {
+		if httpResp != nil {
+			logger.Debug("Error while get your personal token", zap.Error(err))
```
Comment (PatrickMenoti) [inline pkg/api/personal_token/get.go:18]: Error while describing your personal token

File: pkg/api/personal_token/list.go [hunk 4]
```diff
@@ -12,12 +12,16 @@ import (
 
 func (c *Client) List(ctx context.Context) ([]sdk.PersonalTokenResponseGet, error) {
 	logger.Debug("List Personal Tokens")
-
 	resp, httpResp, err := c.apiClient.PersonalTokenApi.ListPersonalToken(ctx).Execute()
 	if err != nil {
-		logger.Error("Error while listing personal tokens", zap.Error(err))
+		if httpResp != nil {
+			logger.Debug("Error while list your personal token", zap.Error(err))
```
Comment (PatrickMenoti) [inline pkg/api/personal_token/list.go:18]: Error while listing your personal token

File: messages/personal_token/messages.go [hunk 5]
```diff
@@ -2,9 +2,9 @@ package personal_token
 
 const (
 	USAGE                      = "personal-token"
-	SHORT_DESCRIPTION_DESCRIBE = "Describe a Personal Token"
-	LONG_DESCRIPTION_DESCRIBE  = "Describe a Personal Token to be used for authentication and security"
-	ASK_PERSONAL_TOKEN_ID      = "Enter the ID of the personal token is linked to:"
-	FLAG_PERSONAL_TOKEN_ID     = "Unique identifier for an personal token"
+	SHORT_DESCRIPTION_DESCRIBE = "Describe a Personal Token's"
```
Comment (PatrickMenoti) [inline messages/personal_token/messages.go:5]: Describe a Personal Token

-----
's is a possessive

File: messages/personal_token/messages.go [hunk 6]
```diff
@@ -2,9 +2,9 @@ package personal_token
...
+	LONG_DESCRIPTION_DESCRIBE  = "Describe a Personal Token's to be used for authentication and security"
+	ASK_PERSONAL_TOKEN_ID      = "Enter the Personal Token's ID:"
+	FLAG_PERSONAL_TOKEN_ID     = "Unique identifier for an Personal Token's"
```
Comment (PatrickMenoti) [inline messages/personal_token/messages.go:8]: Unique identifier for an Personal Token

-----
's is a possessive

**Review by rodrigomaria — APPROVED** [review]

**Review by PatrickMenoti — APPROVED** [review]

**Pull request merged**